src/block-support/supported-blocks.json

```diff
@@ -32,7 +32,8 @@
 		"jetpack/contact-info",
 		"jetpack/email",
 		"jetpack/phone",
-		"jetpack/address"
+		"jetpack/address",
+		"jetpack/tiled-gallery"
```

Comment:
Context for this change https://github.com/wordpress-mobile/gutenberg-mobile/pull/2589